Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -229,7 +247,139 @@ YUI.add('ez-actionbarview-tests', function (Y) {
         },
     });
 
+    rootLocationViewTest = new Y.Test.Case({
+        name: "eZ Action Bar View test for root location",
+
+        setUp: function () {
+            this.locationMock = new Mock();
+            this.content = {};
+            this.contentTypeMock = new Mock();
+
+            Y.eZ.CreateContentActionView = Y.Base.create('createContentActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {}, {
+                contentTypeGroups: {},
+                contentTypeSelectorView: {},
+                contentType: {}
+            });
+            Y.eZ.TranslateActionView = Y.Base.create('translateActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {}, {
+                location: {},
+                content: {}
+            });
+            Y.eZ.MoveContentActionView = Y.Base.create('moveContentActionView', Y.eZ.ButtonActionView, [], {}, {
+                location: {}
+            });
+
+            Mock.expect(this.contentTypeMock, {
+                method: 'hasFieldType',
+                args: ['ezuser'],
+                returns: false
+            });
+
+            Mock.expect(this.locationMock, {
+                method: 'get',
+                args: ['depth'],
+                returns: 1,
+            });
+
+            this.view = new Y.eZ.ActionBarView({
+                location: this.locationMock,
+                content: this.content,
+                contentType: this.contentTypeMock
+            });
+        },
+
+        tearDown: function () {
+            delete Y.eZ.CreateContentActionView;
+            delete Y.eZ.TranslateActionView;
+            delete Y.eZ.MoveContentActionView;
+            this.view.destroy();
+        },
+
+        _isActionDisabled: function (actionId) {
+            var actionFound;
+
+            actionFound = Y.Array.find(this.view.get('actionsList'), function (actionView) {
+                return actionView.get('actionId') === actionId;
+            });
+
+            Assert.isTrue(
+                actionFound.get('disabled'),
+                'The ' + actionId + ' should be disabled'
+            );
+        },
+
+        "Should disable sendToTrashActionView": function () {
+            this._isActionDisabled('sendToTrash', {});
+        },
+    });
+
+    nonRootLocationViewTest = new Y.Test.Case({
```

Comment:
You should merge this test into the previous one. They are two use cases of the previous test.
That would also help to mutualize some code.